docs/schemas/core/IfcProductExtension/Entities/IfcSpatialElement.md

```diff
@@ -3,8 +3,8 @@
 A spatial element is the generalization of all spatial elements that might be used to define a spatial structure or to define spatial zones.
 
 * a hierarchical spatial structure element as _IfcSpatialStructureElement_
-    * a spatial structure is a hiearchical decomposition of the project. That spatial structure is often used to provide a project structure to organize a building project.
-    * A spatial project structure might define as many levels of decomposition as necessary for the building project. Elements within the spatial project structure are site, building, storey, and space or alternatively, site, bridge and bridge part
+    * a spatial structure is a hiearchical decomposition of the project. That spatial structure is often used to provide a project structure to organize a project.
+    * A spatial project structure might define as many levels of decomposition as necessary for the project. Some examples of elements within the spatial breakdown can be: site, building, storey, space. And  for infrastructure projects also: bridge, railway, road, ports, tunnel and parts of them.
```

Comment:
One line starts with small *a*, one with capitalized *A*. It seems that *a* would be consistent.